docs/content/Timeline.md

```diff
@@ -50,28 +50,28 @@ of the child `StyledOcticon` if necessary.
 ```jsx live
 <Timeline>
   <Timeline.Item>
-    <Timeline.Badge bg="red.5">
-      <StyledOcticon icon={FlameIcon} color="white" />
+    <Timeline.Badge bg="prState.closed.bg">
+      <StyledOcticon icon={FlameIcon} color="prState.closed.text" />
     </Timeline.Badge>
-    <Timeline.Body>Red background used when closed events occur</Timeline.Body>
+    <Timeline.Body>Background used when closed events occur</Timeline.Body>
   </Timeline.Item>
   <Timeline.Item>
-    <Timeline.Badge bg="green.5">
-      <StyledOcticon icon={FlameIcon} color="white" />
+    <Timeline.Badge bg="prState.open.bg">
+      <StyledOcticon icon={FlameIcon} color="prState.open.text" />
     </Timeline.Badge>
-    <Timeline.Body>Green background when opened or passed events occur</Timeline.Body>
+    <Timeline.Body>Background when opened or passed events occur</Timeline.Body>
   </Timeline.Item>
   <Timeline.Item>
-    <Timeline.Badge bg="purple.5">
-      <StyledOcticon icon={FlameIcon} color="white" />
+    <Timeline.Badge bg="prState.merged.bg">
+      <StyledOcticon icon={FlameIcon} color="prState.merged.text" />
     </Timeline.Badge>
-    <Timeline.Body>Purple background used when pull requests are merged</Timeline.Body>
+    <Timeline.Body>Background used when pull requests are merged</Timeline.Body>
   </Timeline.Item>
   <Timeline.Item>
-    <Timeline.Badge bg="blue.5">
-      <StyledOcticon icon={FlameIcon} color="white" />
+    <Timeline.Badge bg="icon.info">
+      <StyledOcticon icon={FlameIcon} color="text.info" />
     </Timeline.Badge>
-    <Timeline.Body>Blue background to indicate new events below</Timeline.Body>
+    <Timeline.Body>Background to indicate new events below</Timeline.Body>
```

Comment:
the pr/issue states seemed like they had direct mapping, the info state was a little more tricky. It sticks out like a sore thumb in dark mode.

<img width="460" alt="Screen Shot 2021-03-05 at 11 01 04 AM" src="https://user-images.githubusercontent.com/16739070/110148194-25f63b00-7da2-11eb-9171-d73bbce50ed9.png">

<img width="539" alt="Screen Shot 2021-03-05 at 11 01 12 AM" src="https://user-images.githubusercontent.com/16739070/110148209-2989c200-7da2-11eb-9b51-224e78605810.png">

Reply:
We can remove the info timeline item from this example 👍